### DISCUSSION TITLE
Images handling and data storage

### DISCUSSION THREAD

**Discussion by bruno-luca** [yamcs/yamcs#673] [Q&A] [answered]
Hello everyone. 
I am new to yamcs and i'm having some trouble while implementing some features. I'm able to clone and run [yamcs-quickstart](https://github.com/yamcs/quickstart) repository but i can't figure out two things: 

1. **How can i store incoming data?** I understood how to send my own telemetry to yamcs basing my scripts on the simulator.py that comes with the quickstart, but I do not understand how I can store that incoming data so the next time I launch yamcs application i get all the last data displayed and then start displaying the incoming one.
 
2. **Can i send images from python to yamcs?** I have a python loop that take a photo once every couple of minutes and I need the image to be sent to yamcs for visualization. I have no idea of how to do it.

Any help is welcome!

**Comment by xpromache** [comment, marked as answer]
1. All the data is stored in Yamcs - however when you restart Yamcs it does not remember the last values for all parameters. You can retrieve the data from archive, though. 
2. Yamcs doesn't handle image data, you would have to build that separately somehow.
